File: frontend/src/app/models/passing-test/view-test-dto.ts
import { ViewQuestionDto } from './view-question-dto';

export interface ViewTestDto {
    id: number;
    title: string;
    description: string;
    timeLimitSeconds: string;
    lastModifiedDate: Date;
    authorId: number;

    testQuestions: ViewQuestionDto[];
}

File: QuizApp/src/QuizApp.BLL/Dto/Test/ViewTestDto.cs
using System;
using System.Collections.Generic;
using System.Text;

using QuizApp.BLL.Dto.TestQuestion;

namespace QuizApp.BLL.Dto.Test
{
    public class ViewTestDto
    {
        public int Id { get; set; }

        public string Title { get; set; }

        public string Description { get; set; }

        public TimeSpan TimeLimitSeconds { get; set; }

        public DateTime LastModifiedDate { get; set; }

        public int AuthorId { get; set; }


        public ICollection<ViewQuestionDto> TestQuestions { get; set; } = new List<ViewQuestionDto>();
    }
}
